config/mcp-system/config.yaml

```diff
@@ -0,0 +1,16 @@
+servers:
```

Comment:
Prob don't need this:

`make dev-broker-local` from the other PR will grab config from cluster & rewrite:

https://github.com/kagenti/mcp-gateway/pull/90/files#diff-a42ef31db17cb47677b726741c26c6ffb6afdd5af936255d6744d92f6d97c5c7R69